### PR TITLE
docs: add MAix BiT and Teensy 3.6 to list of supported boards

### PR DESCRIPTION
**Pull request by sago35**
Sipeed MAix BiT
https://github.com/tinygo-org/tinygo/pull/1149

Teensy 3.6
https://github.com/tinygo-org/tinygo/pull/863

### CODE REVIEW

**Comment by sago35** [comment]
Someone needs to write the following
https://github.com/tinygo-org/tinygo-site/issues/96
https://github.com/tinygo-org/tinygo-site/issues/97

File: README.md [hunk 1]
```diff
@@ -75,7 +75,9 @@ The following 35 microcontroller boards are currently supported:
 * [PineTime DevKit](https://www.pine64.org/pinetime/)
 * [Seeed Wio Terminal](https://www.seeedstudio.com/Wio-Terminal-p-4509.html)
 * [Seeed Seeeduino XIAO](https://www.seeedstudio.com/Seeeduino-XIAO-Arduino-Microcontroller-SAMD21-Cortex-M0+-p-4426.html)
+* [Seeed Sipeed MAix BiT](https://www.seeedstudio.com/Sipeed-MAix-BiT-for-RISC-V-AI-IoT-p-2872.html)
 * [SiFIve HiFive1](https://www.sifive.com/boards/hifive1)
+* [SparkFun Teensy 3.6](https://www.sparkfun.com/products/14057)
```
Comment (deadprogram) [inline README.md]: This board is made by PJRC not by Sparkfun:

https://www.pjrc.com/store/teensy36.html

Comment (sago35) [reply README.md]: @deadprogram 
Updated.

**Comment by deadprogram** [comment]
Thank you @sago35 now squash/merging.

**Pull request merged**